patches/minecraft/net/minecraft/loot/LootContext.java.patch

```diff
@@ -26,3 +26,12 @@
        public LootContext.Builder func_216023_a(Random p_216023_1_) {
           this.field_216027_d = p_216023_1_;
           return this;
+@@ -177,7 +189,7 @@
+ 
+       public LootContext func_216022_a(LootParameterSet p_216022_1_) {
+          Set<LootParameter<?>> set = Sets.difference(this.field_216025_b.keySet(), p_216022_1_.func_216276_b());
+-         if (!set.isEmpty()) {
++         if (false && !set.isEmpty()) { // Forge: this prevents mods from adding their own parameters - just ignore extra provided parameters
```

Comment:
This comment makes it sound like this patch prevents mods from adding their own parameters.

Reply:
Yeah, reading that back it could probably be worded a lot cleaner.